commands/describe.go

```diff
@@ -63,14 +64,17 @@ func runDescribe(cmd *cobra.Command, args []string) error {
 		}
 	}
 	gatewayAddress := getGatewayURL(gateway, defaultGateway, yamlGateway, os.Getenv(openFaaSURLEnvironment))
+	cliAuth := NewCLIAuth(token, gatewayAddress)
+	transport := GetDefaultCLITransport(tlsInsecure, &commandTimeout)
+	cliClient := proxy.NewClient(cliAuth, gatewayAddress, transport, &commandTimeout)
 
-	function, err := proxy.GetFunctionInfoToken(gatewayAddress, functionName, tlsInsecure, token, functionNamespace)
+	function, err := cliClient.GetFunctionInfo(context.Background(), functionName, functionNamespace)
```

Comment:
I would create one context `ctx := context.Background` and then use throughout the same command, i think that aligns with the intended usage for context since both these client calls are coming from the same method they come from the same context

Reply:
Agree !! I will make these changes and update it.

Reply:
Updated.